src/ptvsd/wrapper.py

```diff
@@ -1334,16 +1334,36 @@ def _initialize_path_maps(self, args):
 
     def _send_cmd_version_command(self):
         cmd = pydevd_comm.CMD_VERSION
-        default_os_type = 'WINDOWS' if platform.system() == 'Windows' else 'UNIX'
-        client_os_type = self.debug_options.get('CLIENT_OS_TYPE', default_os_type)
-        os_id = client_os_type
-        msg = '1.1\t{}\tID'.format(os_id)
+
+        msg = '1.1\t{}\tID'.format(self._client_os_type)
         return self.pydevd_request(cmd, msg)
 
     @async_handler
     def _handle_launch_or_attach(self, request, args):
         self._path_mappings_received = True
 
+        client_os_type = self.debug_options.get('CLIENT_OS_TYPE', '').upper().strip()
+        if client_os_type and client_os_type not in ('WINDOWS', 'UNIX'):
+            ptvsd.log.warn('Invalid CLIENT_OS_TYPE passed: %s (must be either "WINDOWS" or "UNIX").' % (client_os_type,))
```

Comment:
`warn` is a wrapper around `string.Formatter`, so you can do this:
```py
ptvsd.log.warn('Invalid CLIENT_OS_TYPE passed: {0} (must be either "WINDOWS" or "UNIX").', client_os_type)
```
Note that it must be `{0}` rather than `{}`, because `Formatter` doesn't support the latter on Python 2.

Reply:
Done